deploy-board/deploy_board/settings.py

```diff
@@ -32,9 +32,21 @@
 
 PROJECT_PATH = BASE_DIR
 
-TEMPLATE_DIRS = (
-    os.path.join(BASE_DIR, 'templates'),
-)
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS' : [os.path.join(BASE_DIR, 'templates')],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
```

Comment:
do we really need debug? what is the impact on performance?

Reply:
We do not need debug, only auth is required. I removed the other optional ones. Thanks.